packages/fxa-auth-server/lib/senders/emails/templates/subscriptionReactivation/index.mjml

```diff
@@ -0,0 +1,25 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+
+<mj-section css-class="mb-6">
+  <mj-column>
+    <mj-image src="<%- icon %>" alt="<%- productName %> icon" title="<%- productName %>" width="58px" height="58px"></mj-image>
```

Comment:
I think we may want width/height in a class in case we want to use the same in other templates🤔